andela/rivendell-ah-client
# #160056378 User should be able to rate an articles

#### What does this PR do?
- This PR provides the feature to allow login users to rate an article and view average rating and other raters.

#### Description of Task to be completed?
- allow login users to be able to rate articles.
- allow users to see the average rating of an article.
- allow users to be able to view other users that have rated an article
- allow users to be able to  update their ratings

#### How should this be manually tested?
- Clone the repository
- Run npm install to install all app dependencies
- Run npm run dev to run the start the app in development mode
- Click on an article to navigate to the article's page `/articles/:slug`
- As a login user click on the star icons to rate an article
- Hover your mouse on the `raters` figure view other raters
- Click on any of the raters name or image to see their profiles.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #160056378

#### Screenshots 
![image](https://user-images.githubusercontent.com/39072402/46016386-91655a00-c0cc-11e8-9f2e-5da73c12eadc.png)


![image](https://user-images.githubusercontent.com/39072402/45998216-1420f180-c09a-11e8-89b7-9d414bab92e2.png)

![image](https://user-images.githubusercontent.com/39072402/45998230-1edb8680-c09a-11e8-9df0-17dbcbe255a4.png)
